engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/SelectGameScreen.java

```diff
@@ -163,4 +207,26 @@ private void refreshList(UIList<GameInfo> gameList) {
         gameList.setList(GameProvider.getSavedGames());
     }
 
+    private void loadPreviewImage(GameInfo item) {
+        Texture texture;
+        if (item.getPreviewImage() != null) {
+            TextureData texData = null;
```

Comment:
Another trivial style comment: Unless you're adding clarity somehow avoid overly abbreviating things. `textureData` instead of `texData` adds very few characters yet is entirely clear, while somebody somewhere some day will probably find something else that "tex" might stand for (See: DestSol and excessive abbreviations)